cli/cli/cli.go

```diff
@@ -64,12 +64,14 @@ type CurveAdm struct {
 	memStorage *utils.SafeMap
 
 	// properties (hosts/cluster)
-	hosts               string // hosts
-	clusterId           int    // current cluster id
-	clusterUUId         string // current cluster uuid
-	clusterName         string // current cluster name
-	clusterTopologyData string // cluster topology
-	clusterPoolData     string // cluster pool
+	hosts               string         // hosts
+	disks               string         // disks
+	diskRecords         []storage.Disk // disk
```

Comment:
How about `diskTable`, like `fstab`?

Reply:
`diskTable` refers to a database table that holding records, `diskRecords` refers to the disk records queried from database.